resources/ctl.rb

```diff
@@ -46,16 +46,59 @@
 # end
 
 action :install do
-  packagecloud_repo new_resource.repo do
-    type value_for_platform_family(
-      %w(rhel fedora amazon) => 'rpm',
-      'default' => 'deb'
-    )
+  if node['platform'] != 'windows'
+    packagecloud_repo new_resource.repo do
+      type value_for_platform_family(
+        %w(rhel fedora amazon) => 'rpm',
+        'default' => 'deb'
+      )
+    end
+
+    package 'sensu-go-cli' do
+      action :install
+      version new_resource.version unless new_resource.version == 'latest'
+    end
   end
 
-  package 'sensu-go-cli' do
-    action :install
-    version new_resource.version unless new_resource.version == 'latest'
+  if node['platform'] == 'windows'
+    # This is awaiting a packaged method to be delivered, but provides a resource currently.
+    include_recipe 'seven_zip'
+
+    directory 'c:\temp'
```

Comment:
I don't know enough about windows is this an actual temp dir where it will be blown away on reboot?

Reply:
no, nothing like that, but there is a risk to collision so we can be more specific with the name